cmake/options.cmake

```diff
@@ -52,3 +52,4 @@ option(WITH_STRICT_DATABASE_TYPE_CHECKS "Enable strict checking of database fiel
 set(WITH_SOURCE_TREE    "hierarchical" CACHE STRING "Build the source tree for IDE's.")
 set_property(CACHE WITH_SOURCE_TREE PROPERTY STRINGS no flat hierarchical hierarchical-folders)
 option(WITHOUT_GIT      "Disable the GIT testing routines"                            0)
+option(CONF             "Copy authserver and worldserver .conf.dist files to the project dir"      1)
```

Comment:
`INSTALL_CONF` would make the meaning of the variable more clear.

Reply:
imho better copy_conf xd

Reply:
@ipriver feel free to choose one suggestion ;) As long as it's more than just `CONF`, I'm fine with it.

Reply:
hah, I named it `CONF` because there are `SERVER` and `TOOLS` variables that are used to toggle compiling and I thought it would make sense to have the same one-world variable for configs.
Though if you still want to change the name of it I like `COPY_CONF` more :)

Reply:
Then `COPY_CONF` it is. Makes it definitely easier to search for it.

Reply:
Done. Made a commit amend, rebased to the latest commit in 3.3.5 and force pushed to clean my previous commits in this branch.